catalog_reader/scripts/apps_hashes.py

```diff
@@ -67,22 +68,37 @@ def update_catalog_hashes(catalog_path: str) -> None:
             catalog_base_lib_dir_path = os.path.join(library_dir.as_posix(), lib_version)
             shutil.copytree(catalog_base_lib_dir_path, app_base_lib_dir.as_posix())
 
+            old_version = app_config["version"]
+            if bump_type and app_config['lib_version_hash'] != hashes[lib_version]:
+                if not is_valid_bump_type(bump_type):
```

Comment:
We should validate if bump type is good or not

Reply:
Can you clarify? Isn't `is_valid_bump_type` doing that already?